tendrl/ceph_integration/manager/crud.py

```diff
@@ -32,15 +31,27 @@ def __getattribute__(self, item):
             attr = object.__getattribute__(self, item)
             if callable(attr):
                 def wrap(*args, **kwargs):
-                    LOG.debug("Crud >> %s(%s, %s)" %
-                              (item, args, kwargs))
                     try:
-                        rc = attr(*args, **kwargs)
-                        LOG.debug("Crud << %s" % item)
-                    except Exception:
-                        LOG.exception("Crud !! %s" % item)
-                        raise
-                    return rc
```

Comment:
@brainfunked @shtripat @anmolbabu @nnDarshan @r0h4n @nthomas-redhat LOG.exception, i gave warning priority for this is that ok?